client/me/profile-gravatar/index.jsx

```diff
@@ -25,6 +27,18 @@ export default React.createClass( {
 	render() {
 		const profileURL = `//gravatar.com/${ this.props.user.username }`;
 
+		if ( isEnabled( 'me/edit-gravatar' ) ) {
+			return (
+				<div className="profile-gravatar">
+					<Gravatar user={ this.props.user } size={ 150 } imgSize={ 400 } />
```

Comment:
why is imgSize `400`? might add a comment detailing how that size was derived, or add the derivation as a `const`, like 
```
const GRAVATAR_SIZE = 150;
const GRAVATAR_IMAGE_SIZE = ( GRAVATAR_SIZE * 2 ) + 100; // add 100 because ...
```